tests/FSharp.Compiler.Service.Tests/ConsoleOnlyOptionsTests.fs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+module FSharp.Compiler.Service.Tests.ConsoleOnlyOptionsTests
+
+open System
+open System.IO
+open FSharp.Compiler.Text
+open FSharp.Compiler.AbstractIL.ILBinaryReader
+open FSharp.Compiler.CompilerConfig
+open FSharp.Compiler.CompilerOptions
+open Internal.Utilities
+open NUnit.Framework
+
+// just a random thing to make things work
+let private getBuilder() =
+    TcConfigBuilder.CreateNew(
+        null,
+        FSharpEnvironment.BinFolderOfDefaultFSharpCompiler(None).Value,
+        ReduceMemoryFlag.Yes,
+        Directory.GetCurrentDirectory(),
+        false,
+        false,
+        CopyFSharpCoreFlag.No,
+        (fun _ -> None),
+        None,
+        Range.Zero)
+
+[<Test>]
+let ``Help is displayed correctly`` () =
+    let builder = getBuilder()
+    let blocks = GetCoreFscCompilerOptions builder
+    let fileName = $"{Guid.NewGuid()}"
+    let expectedHelp = File.ReadAllText $"{__SOURCE_DIRECTORY__}/expected-help-output.txt"
+    let print text = File.AppendAllText(fileName, text)
```

Comment:
Do we need to write this in a file? Makes it slower and also it looks like the file isn't cleaned up afterwards.

Reply:
Good point, no files involved now